app/controllers/configuration_controller.rb

```diff
@@ -583,7 +584,7 @@ def timeprofile_update
         end
         return
       end
-      timeprofile = TimeProfile.find(@edit[:timeprofile].id)    # get the current record
+      timeprofile = TimeProfile.find(@edit[:timeprofile_id])    # get the current record
```

Comment:
@ZitaNemeckova should we use @timeprofile that was set in get_form_vars method https://github.com/ZitaNemeckova/manageiq/blob/50d9179f28bf9997b604662a06e69248783637fa/app/controllers/configuration_controller.rb#L407 instead of looking it up again